data/campaigns/The_Hammer_of_Thursagan/scenarios/10_The_Siege_of_Kal_Kartha.cfg

```diff
@@ -223,7 +223,7 @@
                 condition=lose
             [/objective]
             [objective]
-                description= _ "Death of West Gate commander"
+                description= _ "Death of Dulcatulos"
```

Comment:
To me, it tasted like a change made after the initial design. Dulcatulos felt like he came with the addition of S12 and was promoted to S10 and S11 so there would be some reason for him to be there.

My commit comment is exactly my thought when I first hit the scene. And it was odd .. usually we name all the friends like this. And we usually name the enemies and only say "Defeat all enemy leaders" when there is more than one.